ports/gstplugin/servosrc.rs

```diff
@@ -577,6 +611,23 @@ impl BaseSrcImpl for ServoGstSrc {
     }
 
     fn create(&self, src: &BaseSrc, _offset: u64, _length: u32) -> Result<Buffer, FlowError> {
+        // We block waiting for the next frame to be needed.
+        // TODO: Once get_times is in BaseSrcImpl, we can use that instead.
+        // It's been merged but not yet published.
+        // https://github.com/servo/servo/issues/25234
+        let elapsed_micros = self.start.elapsed().as_micros() as u64;
+        let frame_duration_micros = self.frame_duration_micros.load(Ordering::SeqCst);
```

Comment:
If this is the only thread doing reads and writes we can just have Acquire/Release everywhere here, I think

Reply:
actually if it's just one thread then relaxed probably works? unsure.

Reply:
All true, I am being conservative here,m I doubt that the different orderings make much difference, given that the next operations are `thread::sleep` followed by locking a `Mutex`.